### PR TITLE
Budget-App

### PR DESCRIPTION
**Pull request by degisew**
## Budget-App

**In this capstone I implemented the following features:**

- Created Splash screen with Logo and buttons to signup and login.
- Created a login page so that users can log in and access resources.
- Created a signup page so that users can create new accounts.
- Created a category page with a list of categories and a button to add new categories.
- Created a Transaction page with a list of transactions for specific categories and a button to add new ones. 
-  Created unit and integration tests for all the most important components of your RoR application.

shoot

https://user-images.githubusercontent.com/86473715/222891207-80b2baab-ada6-43ab-b4d1-70541c55928a.mp4

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
+  bottom: 0;
+  width: 100%;
+  background-color: #cbc4c4;
+  color: #fff;
+}
+
+nav {
+  background-color: rgba(240, 240, 240);
+  color: #fff;
+}
```
Comment (andersonlebon) [inline app/assets/stylesheets/application.css:12]: - [x] A very fantastic job 👏 however to be consistent with the project requirements, I think you need to follow the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) by using the same layout, colors, margin, space between items...
a simple video review bellow
https://drive.google.com/file/d/1CmSFAMiZoqOacfljR7BXDqZcgNR4uDO2/view?usp=sharing

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @degisew 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

almost-there

## Required changes

- [ ] Kindly make sure in your video presentation you highlight a piece of the code you find interesting or are proud of making. This will make the presentation stronger and sounds more professional. 
image

- [ ] Note that in the requirements It is said that `The project should be deployed and accessible online.` kindly make sure your project is well deployed, you can use render, Heroku,... to do this. and don't forget to add the link to the live version in your README file in the appropriate section.

image

- [ ]  Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (degisew) [reply app/assets/stylesheets/application.css:12]: Thanks for your feedback. I will fix the required changes.

**Review by sumairq — APPROVED** [review]
### Hi @degisew , 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**